app/views/dashboard.html

```diff
@@ -10,6 +10,7 @@
       <p class="performance-select">
         <a ng-class="{active: visMode==='passrate'}" ng-click="setVisMode('passrate')" translate="legend.passrate">Pass Rate (%)</a>
         <a ng-class="{active: visMode==='ptratio'}" ng-click="setVisMode('ptratio')" translate="legend.ptratio">Pupil-Teacher Ratio</a>
+        <a ng-hide="schoolType==='primary'" ng-class="{active: visMode==='gpa'}" ng-click="setVisMode('gpa')" translate="legend.gpa">GPA</a>
```

Comment:
I think `ng-show` for `secondary` schoolType is a little more direct instead of `ng-hide`.